plugin.php

```diff
@@ -115,6 +113,11 @@ function create_initial_json_routes() {
 		}
 
 		$controller->register_routes();
+
+		if ( post_type_supports( $post_type->name, 'custom-fields' ) ) {
+			$meta_controller = new WP_JSON_Meta_Posts_Controller( $post_type->json_base );
```

Comment:
There should probably be a way to override this; I'm thinking `$post_type->json_meta_controller_class`, which defaults to the above?